core/server/services/webhooks/trigger.js

```diff
@@ -0,0 +1,76 @@
+const _ = require('lodash');
+const common = require('../../lib/common');
+const models = require('../../models');
+const pipeline = require('../../../server/lib/promise/pipeline');
+const request = require('../../../server/lib/request');
+
+function updateWebhookTriggerData(id, data) {
+    models.Webhook.edit(data, {id: id}).catch(() => {
+        common.logging.warn(`Unable to update last_triggered for webhook: ${id}`);
+    });
+}
+
+function makeRequests(webhooksCollection, payload, options) {
+    _.each(webhooksCollection.models, (webhook) => {
+        const event = webhook.get('event');
+        const targetUrl = webhook.get('target_url');
+        const webhookId = webhook.get('id');
+        const reqPayload = JSON.stringify(payload);
+
+        common.logging.info('webhook.trigger', event, targetUrl);
+        let triggeredAt = Date.now();
+        request(targetUrl, {
+            body: reqPayload,
+            headers: {
+                'Content-Length': Buffer.byteLength(reqPayload),
```

Comment:
Was gonna suggest and edit here but after reading up a bit more on 'Content-Lenght' calculation seems like we have to make a change in our frame instead :smiley: 

@kirrg001 seems like there's an improvement to be done it in frames' [headers stage](https://github.com/TryGhost/Ghost/blob/master/core/server/api/shared/headers.js#L24)

Because 'Content-Lenght' is:
> The Content-Length entity-header field indicates the size of the entity-body, in decimal number of OCTETs
(https://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html)

And stings `.lenght` does return exactly the byte size in bytes :)

Reply:
@gargol does this require a change in the PR or is this comment referencing something else?

Reply:
@kevinansfield  something else, but here's a good use case to refer to later :)

Reply:
> Buffer.byteLength(reqPayload)

Yeah sounds correct 👍 
Could you pls raise a separate issue? Thanks 👍